### PR TITLE
Test some custom methods from StringUtils and MainUtils classes.

### PR DESCRIPTION
**Pull request by emiryazicii**
Test add() and frequency() methods.

### CODE REVIEW

**Review by drnajimov — COMMENTED** [review]
Sorry I didn't get that what we need to do on this commit

File: src/utils/Tests.java [hunk 1]
```diff
@@ -2,13 +2,25 @@
...
 
-    // tc145
+        System.out.println(result);
 
+        // test frequency() method from StringUtils class.(frequency of char)
+
+        int result2 = StringUtils.frequency("JAVANARAN", 'A');
+
+        System.out.println(result2);
+
```
Comment (drnajimov) [inline src/utils/Tests.java:19]: String str = "JAVANARAN";
        char ch = 'A';

        int result2 = 0;

        for (int i = 0; i < str.length(); i++) {

            if( str.charAt(i) == ch ){
               
                result2++;
            }

        }


        System.out.println(result2);

**Review by jika17 — APPROVED** [review]
Thanks for updates!

**Comment by emiryazicii** [comment]
> Sorry, I didn't get that what we need to do on this commit. Do we need to add that method and solve the problem?

You just need to review and approve the changes.
I already tested some methods.

**Comment by emiryazicii** [comment]
> Thanks for updates!

No problem Jika, I see that you deleted emir branch after merge. It's ok but you don't need to do that for the future requests.

**Comment by drnajimov** [comment]
I try to approve that but it's gone

**Comment by drnajimov** [comment]
I guess it's merged automatically

**Comment by emiryazicii** [comment]
> I guess it's merged automatically

Jika approved that one.

**Pull request merged**